vendor/k8s.io/client-go/util/cert/BUILD

```diff
@@ -25,9 +25,10 @@ go_library(
         "io.go",
         "pem.go",
     ],
-    data = [
-        "testdata/dontUseThisKey.pem",
-    ],
+    # Hand editted out, as this doesn't seem to be required.
```

Comment:
This change was overwritten by `hack/update-deps.sh`, so if it needs to stick around it should be automated.

Reply:
Done, this is changed in `./hack/update-deps.sh` now.